packages/storybook/src/nimble/tabs/tabs.stories.ts

```diff
@@ -42,18 +44,58 @@ const metadata: Meta<TabsArgs> = {
     }
 };
 
+const simpleTabs: readonly TabArgs[] = [
```

Comment:
when using `as const` we should avoid overriding that more specific type with a manual weaker type
```suggestion
const simpleTabs = [
```